From 29221c49aab6993cef35b356b0aa770e5d14355e Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Mon, 10 Sep 2018 17:05:37 +0200
Subject: [PATCH 0685/1051] arm64: dts: marvell: remove phy-names properties
 from armada 7k/8k/8kp

Because the pcie-armada8k driver no longer relies on phy-names properties,
remove them from all related dts'es.

Change-Id: I7f50ad5a572e5daa4560b31d02d69dae485b8e8c
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-B.dts  | 2 --
 arch/arm64/boot/dts/marvell/armada-7040-db-C.dts  | 2 --
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts  | 2 --
 arch/arm64/boot/dts/marvell/armada-7040-db-E.dts  | 2 --
 arch/arm64/boot/dts/marvell/armada-7040-db.dts    | 1 -
 arch/arm64/boot/dts/marvell/armada-8040-db-B.dts  | 5 -----
 arch/arm64/boot/dts/marvell/armada-8040-db-C.dts  | 5 -----
 arch/arm64/boot/dts/marvell/armada-8040-db-D.dts  | 5 -----
 arch/arm64/boot/dts/marvell/armada-8040-db-E.dts  | 5 -----
 arch/arm64/boot/dts/marvell/armada-8040-db.dts    | 5 -----
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 1 -
 arch/arm64/boot/dts/marvell/armada-8082-db-B.dts  | 2 --
 arch/arm64/boot/dts/marvell/armada-8082-db.dts    | 2 --
 arch/arm64/boot/dts/marvell/armada-8084-db.dts    | 2 --
 arch/arm64/boot/dts/marvell/armada-8162-db.dts    | 2 --
 arch/arm64/boot/dts/marvell/armada-8164-db.dts    | 3 ---
 16 files changed, 46 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
index 0e40e10993ed..805d9e16809f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
@@ -43,13 +43,11 @@
 	status = "okay";
 	num-lanes = <4>;
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_usb3_1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
index b3670d00a250..f8ed7012c08e 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
@@ -50,13 +50,11 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index 45c2602da9ea..4243053b1392 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -75,13 +75,11 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
index 14af17dbd11f..c02e7e039bb5 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
@@ -42,13 +42,11 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 3741d0e37516..3480d8337477 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -46,7 +46,6 @@
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_i2c0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
index 22b9585cc319..4638e2bb955c 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
@@ -45,14 +45,12 @@
 	num-lanes = <4>;
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP0 expansion */
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON9 on CP0 expansion */
@@ -91,21 +89,18 @@
 &cp1_pcie0 {
 	status = "okay";
 	phys = <&cp1_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON7 on CP1 expansion */
 &cp1_pcie1 {
 	status = "okay";
 	phys = <&cp1_comphy4 1>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
 	phys = <&cp1_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON4 on CP1 expansion */
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
index 559ff670085a..95bfad18f08b 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
@@ -45,14 +45,12 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP0 expansion */
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
@@ -116,21 +114,18 @@
 &cp1_pcie0 {
 	status = "okay";
 	phys = <&cp1_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON7 on CP1 expansion */
 &cp1_pcie1 {
 	status = "okay";
 	phys = <&cp1_comphy4 1>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
 	phys = <&cp1_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON4 on CP1 expansion */
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
index 79ce3c6d4069..0e0871dccead 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
@@ -44,14 +44,12 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP0 expansion */
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
@@ -116,21 +114,18 @@
 &cp1_pcie0 {
 	status = "okay";
 	phys = <&cp1_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON7 on CP1 expansion */
 &cp1_pcie1 {
 	status = "okay";
 	phys = <&cp1_comphy4 1>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
 	phys = <&cp1_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON4 on CP1 expansion */
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
index 8191da3d5311..a3199682d936 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
@@ -66,14 +66,12 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP0 expansion */
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_sata0 {
@@ -138,21 +136,18 @@
 &cp1_pcie0 {
 	status = "okay";
 	phys = <&cp1_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON7 on CP1 expansion */
 &cp1_pcie1 {
 	status = "okay";
 	phys = <&cp1_comphy4 1>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
 	phys = <&cp1_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON4 on CP1 expansion */
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 3b42efbf9c72..3dc551796b55 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -40,14 +40,12 @@
 &cp0_pcie0 {
 	status = "okay";
 	phys = <&cp0_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP0 expansion */
 &cp0_pcie2 {
 	status = "okay";
 	phys = <&cp0_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp0_i2c0 {
@@ -114,21 +112,18 @@
 &cp1_pcie0 {
 	status = "okay";
 	phys = <&cp1_comphy0 0>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON7 on CP1 expansion */
 &cp1_pcie1 {
 	status = "okay";
 	phys = <&cp1_comphy4 1>;
-	phy-names = "pcie-phy0";
 };
 
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
 	phys = <&cp1_comphy5 2>;
-	phy-names = "pcie-phy0";
 };
 
 &cp1_i2c0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 6710ef89f704..13022d84c6c5 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -185,7 +185,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8082-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8082-db-B.dts
index fad6c46b402d..e2ba0ba565b2 100644
--- a/arch/arm64/boot/dts/marvell/armada-8082-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8082-db-B.dts
@@ -128,7 +128,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_pcie1 {
@@ -195,7 +194,6 @@
 		&cp1_comphy1 0
 		&cp1_comphy2 0
 		&cp1_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp1_pcie1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8082-db.dts b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
index db744a0c9d47..6b9a2102a228 100644
--- a/arch/arm64/boot/dts/marvell/armada-8082-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
@@ -115,7 +115,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_pcie1 {
@@ -186,7 +185,6 @@
 		&cp1_comphy1 0
 		&cp1_comphy2 0
 		&cp1_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp1_pcie1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8084-db.dts b/arch/arm64/boot/dts/marvell/armada-8084-db.dts
index 16bab0e6d81e..23559485b928 100644
--- a/arch/arm64/boot/dts/marvell/armada-8084-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8084-db.dts
@@ -133,7 +133,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_pcie1 {
@@ -204,7 +203,6 @@
 		&cp1_comphy1 0
 		&cp1_comphy2 0
 		&cp1_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp1_pcie1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8162-db.dts b/arch/arm64/boot/dts/marvell/armada-8162-db.dts
index 82364d49f9cf..0baddf49e1ac 100644
--- a/arch/arm64/boot/dts/marvell/armada-8162-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8162-db.dts
@@ -103,7 +103,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_i2c0 {
@@ -136,7 +135,6 @@
 		&cp4_comphy1 0
 		&cp4_comphy2 0
 		&cp4_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp4_pcie1 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8164-db.dts b/arch/arm64/boot/dts/marvell/armada-8164-db.dts
index 483133e7bfd4..f6bc7e65c7e5 100644
--- a/arch/arm64/boot/dts/marvell/armada-8164-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8164-db.dts
@@ -115,7 +115,6 @@
 		&cp0_comphy1 0
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp0_pcie1 {
@@ -145,7 +144,6 @@
 		&cp1_comphy1 0
 		&cp1_comphy2 0
 		&cp1_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp1_pcie1 {
@@ -175,7 +173,6 @@
 		&cp4_comphy1 0
 		&cp4_comphy2 0
 		&cp4_comphy3 0>;
-	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
 &cp4_pcie1 {
-- 
2.17.1

